metadata-ingestion/src/datahub/ingestion/api/source_helpers.py

```diff
@@ -89,9 +91,18 @@ def auto_status_aspect(
         else:
             raise ValueError(f"Unexpected type {type(wu.metadata)}")
 
+        if (
+            not isinstance(wu.metadata, MetadataChangeEventClass)
+            and wu.metadata.entityType in skip_entities
+        ):
+            # If any entity does not support aspect 'status' then skip that entity from adding status aspect.
```

Comment:
we have a map of exactly what entity types support what aspects - can we look this information up there instead?

Reply:
I can provide some pointers on how to do this

Reply:
Pls provide some pointers on this

Reply:
Use the helper method from here https://github.com/datahub-project/datahub/pull/9120